instrumentation/opentelemetry-instrumentation-grpc/tests/test_server_interceptor.py

```diff
@@ -106,12 +106,12 @@ def handler(request, context):
             self.assertIs(span.kind, trace.SpanKind.SERVER)
 
             # Check version and name in span's instrumentation info
-            self.check_span_instrumentation_info(
+            self.assertSpanInstrumentationInfo(
```

Comment:
Nit: I would rename this `asserSpanHasInstrumentationInfo` to make it more consistent